src/Transport/Topology/Topologies/ForwardingTopologySectionManager.cs

```diff
@@ -102,28 +112,28 @@ public TopologySection DeterminePublishDestination(Type eventType)
                     BuildTopicBundles(namespaces, addressingLogic);
                 }
 
-                return new TopologySection()
+                return new TopologySection
                 {
-                    Entities = SelectSingleRandomTopicFromBundle(topics),
+                    Entities = SelectFirstTopicFromBundle(topics),
                     Namespaces = namespaces
                 };
             });
         }
 
-        IEnumerable<EntityInfo> SelectSingleRandomTopicFromBundle(List<EntityInfo> entityInfos)
+        IEnumerable<EntityInfo> SelectFirstTopicFromBundle(List<EntityInfo> entityInfos)
         {
-            var index = randomGenerator.Next(0, entityInfos.Count);
+            const int index = 1;
```

Comment:
Should be index 0 as we want to route everything through topic called "bundle-1".